src/convection/incflo_compute_MAC_projected_velocities.cpp

```diff
@@ -22,14 +22,46 @@ incflo::compute_MAC_projected_velocities (
     // We first compute the velocity forcing terms to be used in predicting
     //    to faces before the MAC projection
     if (m_advection_type != "MOL") {
-
-        if (m_godunov_include_diff_in_forcing)
-            for (int lev = 0; lev <= finest_level; ++lev)
-                MultiFab::Add(*vel_forces[lev], m_leveldata[lev]->divtau_o, 0, 0, AMREX_SPACEDIM, 0);
+        
```

Comment:
The advect_momentum flag does not apply in computing MAC velocities; we only ever extrapolate velocities.

Reply:
It is true that only velocities are extrapolated to the faces. But the dimensions of vel_forces and divtau_o are inconsistent here when advect_momentum flag is true. While vel_forces is always [cm/s^2], divtau_o is [gm/cm^2s^2] when advect_momentum is true. Therefore, divtau needs to be divided by \rho before adding to vel_forces when advect_momentum is true. It is fine when advect_momentum is false.

Reply:
In DiffusionTensorOp.cpp and DiffusionScalarOp.cpp (at the very bottom of each file) I see logic that multiplies divtau by (1/rho) when (!advect_momentum).

So here since it's always vel that is extrapolated to faces, the divtau still needs this scaling if advect_momentum=T - I think this is ok.